apps/mobile/mobile-app/components/auth/benefits-section.tsx

```
@@ -0,0 +1,114 @@
+      {/* Impact Stats */}
+      <HStack gap={12} className="mb-6">
+        <ImpactStat value="100+" label="Verified Projects" />
+        <ImpactStat value="$1.7B" label="Available Regional Funding" />
+      </HStack>
+
+      {/* Benefit Cards */}
+      {isMobile ? (
+        <VStack gap={12}>
+          {benefits.map((benefit, index) => (
+            <Motion.View
+              key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)